.github/workflows/publish_connectors.yml

```diff
@@ -32,9 +32,41 @@ on:
         description: "URL to the airbyte-ci binary to use for the action. If not provided, the action will use the latest release of airbyte-ci."
         default: "https://connectors.airbyte.com/airbyte-ci/releases/ubuntu/latest/airbyte-ci"
 jobs:
+  list_connectors:
+    name: List connectors to publish
```

Comment:
this is mostly copied from connector-ci-checks.yaml. Main differences are
* this also has the "if manual run, use `${{ inputs.connectors-options }}` thing"
* and the output variables are different